pkg/client/kubeclient.go

```diff
@@ -138,7 +138,6 @@ func (nc SchedulerKubeClient) Create(pod *v1.Pod) (*v1.Pod, error) {
 }
 
 func (nc SchedulerKubeClient) Delete(pod *v1.Pod) error {
-	// TODO make this configurable for pods
```

Comment:
Could you please move this TODO to jira?

Reply:
Oh yes, I had reported a jira. https://issues.apache.org/jira/browse/YUNIKORN-2734